### ISSUE TITLE
B902 false positive: positional-only arguments in class methods

### ISSUE THREAD

**Issue by bryanforbes** [PyCQA/flake8-bugbear#157]
The following code triggers B902 on the `bar` method:

```python
class Foo:
    def bar(self, a, /):
        pass
```

**Comment by cooperlees** [comment]
Thanks for the report.

If someone has the cycles, would definitely accept a unit-tested PR that ignores this appropriately. And I guess the test will have to be skipped until >= 3.9?

**Comment by bryanforbes** [comment]
It looks like on Python 3.8+, it should be doing `node.args.posonlyargs + node.args.args`. I'll submit a PR later today.

**Comment by bryanforbes** [comment]
What would it take to get a new release with this fix in it?

**Comment by cooperlees** [comment]
Just time for me, I’ll try today / next few days. It’s just making sure CHANGES are up to date in the readme that takes time. The release is automated once I cut a github release. So if you have cycles to help double check that and do a PR finish that then it takes me a lot less time.

**Comment by cooperlees** [comment]
You also have `pip install git+git://github.com/PyCQA/flake8-bugbear` Support to install master.

**Comment by bryanforbes** [comment]
> So if you have cycles to help double check that and do a PR finish that then it takes me a lot less time.

Done: https://github.com/PyCQA/flake8-bugbear/pull/160

**Comment by cooperlees** [comment]
https://github.com/PyCQA/flake8-bugbear/actions/runs/617988022
- Released. Thanks for the nudge and help!
